addons/account/tests/test_chart_template.py

```diff
@@ -725,3 +728,63 @@ def local_get_data(self, template_code):
             'translation.test_chart_template_company_test_free_tax.name@en_US': 'Free Tax',
             'translation.test_chart_template_company_test_free_tax.name@fr_BE': 'Free Tax FR',
         })
+
+    def _get_csv_data(self, template_code, model, module):
```

Comment:
I accept your suggestion but I do not agree 100% with it.

It's true that one includes the other, but one is a unit test on parse_csv only, the other is an integration test that gives the loading of tax data for granted.

Since we're always in time to decouple tests if and when we'll decouple the two things, I'm anyway favorable.

Reply:
The semplification is very nice, thanks and congrats as always.